lib/backends/system-manager-backend.js

```diff
@@ -49,13 +51,24 @@ class SystemManagerBackend extends KVBackend {
       client = this._clientFactory(factoryArgs)
     }
     try {
-      const data = await client
-        .getParameter({
+      const check = await client
+        .getParameterHistory({
```

Comment:
`getParameterHistory()` does not return all results, it returns at max 50. Using `NextToken` you can fetch more.

Reply:
Thanks for your answer, i'm really excited of contributing to the project.

The getParameterHistory() is a free API call if you use Standard parameter. (https://aws.amazon.com/systems-manager/pricing/#Parameter_Store)

I'm adding e2t tests and using NextToken option. I will push the changes soon. Thanks for your review.

Reply:
Hi @moolen i've updated the PR adding e2e test (with 500 parameters because when i tested with 1K it gives me TimeOutError).

I also added NextToken to the request so i can fetch more secret.

Thanks